markdown/explore-dmr/explore-dmr.md

```diff
@@ -0,0 +1,295 @@
+Duration: 0:02:00
+
+The Solace Try Me Tab is small CodePen application which was created to provide a simple way of publishing and subscribing to Solace topics and queues.  The application produces and consumes WebSocket (WS) events to Solace PubSub+ brokers over a network.  Thus is it not required that the Try Me tool be used to connect to a local broker, however the tooling is setup to make connecting to the "local" Solace Broker effortless. Using the Try Me tool you can explore the following Solace features:
+* Publish Direct Messages
+* Publish Persistent Messages
+* Publish to topics
+* Publish directly to queues
+* Subscribe to Solace topics
+* Subscribe to Solace queues
+* Subscribe to Solace topics using Solace wild cards
+* Experience Solace Broker translate different message protocols (requires the use of Try Me and an additional publisher or subscriber).
+
+<br>
+
+**In the next section we will explore how using the Try Me tool to Publish and Subscribe only to a single individual Solace Broker will allow us to obtain events from any accessible topic published to any of the Solace Brokers connected to our Solace DMR Event Mesh.** 
+<br>🆘!!!Maybe here I should include some topic architecture for 2 brokers that will allow you to see the DMR Event Mesh in action?
+
+## Setting up and running "Try Me"
+Duration 00:10:00
+
+**Option A: For your local (on-premise) broker**
+
+1. Navigate to your on-premise broker WebUI and Login
+    
+    ![Untitled](img/Untitled.png)
+    
+2. Select the Message VPN you wish to use
+    
+    ![Untitled](img/Untitled_1.png)
+    
+3. Select“Try Me” tab on the left side bar menu inside of the WebUI
+    
+    ![Untitled](img/Untitled_2.png)
+    
+4. Populate with your on-premise broker details into the Publisher side, unless you are running the broker on your local machine you will likely have to change the “Broker URL”, “Client Username” and “Client Password”. You may also have to change the “Message VPN” depending on your setup. 
+    
+    ![Untitled](img/Untitled_3.png)
+    
+5. Select the “Connect” button on the Publisher side of “Try Me”.
+    
+    ![Untitled](img/Untitled_4.png)
+    
+6. Select the “Connect” button on the Subscriber side of “Try Me” as well.  The connection details will carry over from the Publisher side unless you expand the drop down menu, uncheck the “Same as Publisher” option and modify them.
+    
+    ![Untitled](img/Untitled_5.png)
+    
+7.  Now we can customize our Publish topic, Delivery Mode and Message Content and Publish messages directly to our local (on-premise) broker. 
+    
+    ![Untitled](img/Untitled_6.png)
+    
+8. We can also subscribe to any topics that we would like on the right side by changing the subscribe field and pressing “Subscribe”. The topic can be one that is published to our local broker or published to any other broker in the DMR(Event Mesh) that your user has access to.
+Do not forget about our Solace wild cards here 
```

Comment:
I recommend linking to https://docs.solace.com/Messaging/Wildcard-Charaters-Topic-Subs.htm here